### ISSUE TITLE
[BUG]:”Please Sign In” Error when trying to play any videos

### ISSUE THREAD

**Issue by alien8tor** [yuliskov/SmartTube#4233]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

25.08

### Device Type

Smart TV/Box

### Affected Android

Android 12

### Steps to reproduce the bug

I’ve tried on other devices nvidia shield tv with same 25.08 build and videos don’t play only with message “please sign in”

### Actual behavior

Just noticed when I load up smart tube can’t play any videos all videos say “Please Sign In” it was working fine yesterday but when I tired today no videos are playing, am on 25.08 build.  On google tv, is any one else experiencing this issue as well

### Additional information

Google TV chromecast

**Comment by doughaase** [comment]
Same here, but with FireOS (FireTv Stick). Can't play a lot of videos.

**Comment by seanhumes1002** [comment]
Reporting the same issue on my Smart TV running Android 11.

**Comment by NikeLaosClericus** [comment]
Same issue, not all videos, but most.
Nvidia Shield Pro.

**Comment by cadixu** [comment]
Same issue for three days, not all videos.
for example : https://www.youtube.com/live/E0zhe2gkXBs?si=YZpUbfYLUP2oaj4s
this live videos can watch on original youtube app and web, but cannot watch on SmartTube

**Comment by AZDarkknight** [comment]
Same issue on fire stick - still happening at 12/23 5.45am PST.

**Comment by Fannangir** [comment]
Same here :(.

**Comment by wolfpurplemoon** [comment]
yep getting this issue with more and more videos over the last few days

**Comment by ardunster** [comment]
Same, 25.08 on Nvidia Shield. For me it seems to mostly be the first video I click. It hangs on a screen showing the thumbnail and lists the title of the video as "Please sign in" for several seconds, then automatically loads the next video (which I don't want at all, there is no reason to assume the next video in the list is one I want to watch now just because one failed).

**Comment by ardunster** [comment]
For those of you getting the error on most/all videos, are you signed in? I've noticed YouTube in the past week or two has not been letting me access anything in private tabs or not signed in on any device. I am signed in in smarttube though.

**Comment by ardunster** [comment]
Looks like this is probably  actually a duplicate of this older issue https://github.com/yuliskov/SmartTube/issues/4227

**Comment by ongubsy** [comment]
> For those of you getting the error on most/all videos, are you signed in? I've noticed YouTube in the past week or two has not been letting me access anything in private tabs or not signed in on any device. I am signed in in smarttube though.

I only use it already signed in.

**Comment by AZDarkknight** [comment]
> For those of you getting the error on most/all videos, are you signed in? I've noticed YouTube in the past week or two has not been letting me access anything in private tabs or not signed in on any device. I am signed in in smarttube though.

No, I dont sign in on smarttube due not wanting my email to be vulnerable. Ive never had the problem until a few days ago.

**Comment by ram-sez** [comment]
this just started yesterday for me. It seems to be almost entirely music videos.

**Comment by sizzlik** [comment]
Yep..same here on FireTV. Only one channel affected for me so far..like ram-sez above me said, its a music reaction channel for me.

**Comment by AZDarkknight** [comment]
The channels it is affecting for me is every one that I have looked up, so its a Starcraft 2 channel, dayz, 2 risk channels, a game show channel and a cooking channel. I havent gone beyond looking at random channels as it seems nothing will load, just goes to sign in then skips to the next, rinse and repeat.

**Comment by ongubsy** [comment]
haven't noticed a pattern: there's vlogs, gaming channels, music clips... a lot of them affected.

**Comment by Union-Carbide** [comment]
Happening to me as well as of just a few moments ago

Android TV OS 11
Security patch Aug 1 2024
Kernel
4.19.182-ab571
#1 Wed Oct 16 16:32:02 CST 2024
Android TV OS build RTT2.211209.001

URLs recently affected:
https://m.youtube.com/watch?v=3ryk45IYJpk
https://m.youtube.com/watch?v=UInDr3_XOAQ

**Comment by beastchin** [comment]
同样的问题，出现在音乐类，时政类可以播放

**Comment by Nabukodonosor** [comment]
Same here on Fire Stick 4k Max.

**Comment by konlanx** [comment]
I encountered this issue on a FireTV Stick 4K 2 and was able to resolve it by uninstalling and installing the app following the guides on the homepage.

**Comment by jdtechsol** [comment]
Just started seeing this issue today, Fire Stick 4K 2, will try reinstall

**Comment by tbellomy** [comment]
^^ switching to the beta worked for me. Thanks!

**Comment by Nabukodonosor** [comment]
> ^^ switching to the beta worked for me. Thanks!

Is the Notifications screen working for you?

**Comment by Nabukodonosor** [comment]
https://prnt.sc/Dj7-_hdNjE_k

**Comment by yuliskov** [comment]
It's a known issue. This happened because of YouTube's protection mechanism. I've addressed the issue in the beta.

**Comment by sizzlik** [comment]
This fight will never end it seems. i had the problem a few weeks ago, downgraded to another version as others suggested wich fixed it..few days later an update came..installed, no problems. But since yesterday its back and this time more annoying than before (for me)..really every 2nd video is please sign in..no matter what type of vid, no matter if searched for, recommended or next video to play, new or old..really annoying. I know its google and not smarttube..but damn its annoying..why is this ultra rich, internet dominating corp so keen to die on that hill? If those "smart" people over there should have learned anything over the past 3-4 decades its 1.) Hackers&Crackers always win. Every new lock is just a new game for a hive mind to be disassembled. and 2nd) Fekkin Streisand effect..announcing the war on adblockers just hypes up coders lookin for a challenge. Too bad geohot is now on the "good" side. Anyways..lookin forward to the next patch. -Rant over- Thanks yuliskov & contributers for this app..you do a great job.

**Comment by yuliskov** [comment]
@sizzlik how does the app behave when the "sign in" error is encountered? Is it reboots or switching to the next video?

**Comment by Nabukodonosor** [comment]
> It's a known issue. This happened because of YouTube's protection mechanism. I've addressed the issue in the beta.

Addressed as in solved? Because it doesn't work in beta either.

**Comment by yuliskov** [comment]
@Nabukodonosor yes. Try this option if you still experience this issue. Settings/Player/Dev. options/Playback buffering fix

**Comment by Nabukodonosor** [comment]
> [@Nabukodonosor](https://github.com/Nabukodonosor) yes. Try this option if you still experience this issue. Settings/Player/Dev. options/Playback buffering fix

Nope, didn't work.

**Comment by sizzlik** [comment]
> [@sizzlik](https://github.com/sizzlik) how does the app behave when the "sign in" error is encountered? Is it reboots or switching to the next video?

For me on an old fire vt stick it just stops. Like the video wats to start with the play menu accessable just nothing happens when pressing play and where the video title should be its just please sign in. No reboot, no next video (my video is set to continious play). weird little thing i noticed is that some vids suddenly dont have that problem anymore when they had it a day before..but not all, just like 2/10 happen to work again later and some videos that just came out and worked didnt work some time later on a rewatch try. Doesnt matter the type of video, from big channels to nobodys.

**Comment by dudleyscud** [comment]
This has happened on and off for me on my Shield TV. I went into the developer settings and checked a few of them that might help fix it. Not sure which ones. Worked straight away after restarting the app with new settings.

**Comment by wolfpurplemoon** [comment]
I've had this issue on and off on two different TVs and more than one youtube account for weeks, but it seems to have been happening on about half the videos I've tried today and no amount of restarting has helped

**Comment by dudleyscud** [comment]
> I've had this issue on and off on two different TVs and more than one youtube account for weeks, but it seems to have been happening on about half the videos I've tried today and no amount of restarting has helped

Have you tried to change settings in Settings>Player>Developer Options?

I turned on some of those and it prompted be to restart app for the settings to take effect and it worked for be straight away.
Settings I have checked to be on are
Premium fix
Playback buffering fix
Prefer IPv4 DNS
Audio sync fix
Unlock high nitrate 1080 vp9
Unlock high nitrate mp4a

**Comment by guitarnut1** [comment]
Thanks for sharing that. I don't see that "Premium fix" on my Amazon fire tv, beta, 25.66. Works fine.

**Comment by Adlopa** [comment]
Same issue today on FireStick 4K with v25.87.

**Comment by rpgdev** [comment]
I'm facing the same issue with latest stable (25.87 I believe). So far it's only with one creator, and only with her latest livestreams, not past ones even though I faced the same issue with the past livestreams when they were the latest one.

**Comment by sizzlik** [comment]
it started to become a roallercoaster. Just now i searched for something and pretty much every 2nd vid was "please sign in", i then restarted my firetv stick and suddenly the vid that didnt work before, works now. Its weird and got no real pattern to find the root of the problem

**Comment by yuliskov** [comment]
I'm trying to fix in this build. Anyone willing to test? Installed over existed beta.

https://github.com/yuliskov/SmartTube/releases/download/latest/SmartTube_beta_testing.apk

**Comment by GentlemanSauer** [comment]
> I'm trying to fix in this build. Anyone willing to test? Installed over the existed beta.
> 
> https://github.com/yuliskov/SmartTube/releases/download/latest/SmartTube_beta_testing.apk

Was having this issue after a whole video played, I couldn't rewatch it. It would get stuck on the controls and say Please Sign In. 

I just tested this beta apk and tested with the 2 videos I had issues with. They both play now signed in and signed out.

Insignia FireTV 2019
FireOS 6.7.0.8 (NS6708/5570)

Was using newest stable version of STN when issue arose.

**Comment by yuliskov** [comment]
@GentlemanSauer, thanks for the feedback. Let me know if case something changes.  
A reminder for other participants. Guys, please test the possible fix  https://github.com/yuliskov/SmartTube/issues/4233#issuecomment-2650742625

**Comment by joximijanum** [comment]
> [@GentlemanSauer](https://github.com/GentlemanSauer), thanks for the feedback. Let me know if case something changes. A reminder for other participants. Guys, please test the possible fix [#4233 (comment)](https://github.com/yuliskov/SmartTube/issues/4233#issuecomment-2650742625)

Works bro, good job!

**Comment by MYusufY** [comment]
Same issue here :/

**Comment by ppavacic** [comment]
> I'm trying to fix in this build. Anyone willing to test? Installed over the existed beta.
> 
> https://github.com/yuliskov/SmartTube/releases/download/latest/SmartTube_beta_testing.apk

Works, thanks!

**Comment by guitarnut1** [comment]
Does the latest beta version update 25.97 have this fix?

**Comment by daby0729** [comment]
On mine I went to settings then I went to player I went to developer options scroll all the way down I have checked on unlock all video formats I checked off Force SW video decoder I checked off frame drop fix number 1 and frame drop fixed number 2 and I'm good to go... yeah it was driving me crazy I've had to look it up it's been going on for several weeks I'm glad I finally figured it out hope this helps you guys let me know

**Comment by digger78** [comment]
> On mine I went to settings then I went to player I went to developer options scroll all the way down I have checked on unlock all video formats I checked off Force SW video decoder I checked off frame drop fix number 1 and frame drop fixed number 2 and I'm good to go... yeah it was driving me crazy I've had to look it up it's been going on for several weeks I'm glad I finally figured it out hope this helps you guys let me know

This doesn't make any difference to me.  I'm stilling getting the sign in error on some videos and not on others.

**Comment by yuliskov** [comment]
Published the beta with the latest fixes.
https://github.com/yuliskov/SmartTube/releases/tag/26.07

**Comment by DreadfulDrummer** [comment]
I just started getting this problem today, installed the beta, opened it and then immediately closed it to check on the stable build again and it didn't have the problem anymore. Technology is so funny.

**Comment by daniushko** [comment]
Same started with me yesterday. It was one channel initially with more joining.
Android tv app.

**Comment by rpgdev** [comment]
Updated to 26.11 and it's *worse* now. I still get "Sign in to confirm you're not a bot" on the same channel _and_ get 403 error on every other video :(

**Comment by Heroshi1947** [comment]
> On mine I went to settings then I went to player I went to developer options scroll all the way down I have checked on unlock all video formats I checked off Force SW video decoder I checked off frame drop fix number 1 and frame drop fixed number 2 and I'm good to go... yeah it was driving me crazy I've had to look it up it's been going on for several weeks I'm glad I finally figured it out hope this helps you guys let me know

hey sir this tweaking worked for me thanks

**Comment by rpgdev** [comment]
> On mine I went to settings then I went to player I went to developer options scroll all the way down I have checked on unlock all video formats I checked off Force SW video decoder I checked off frame drop fix number 1 and frame drop fixed number 2 and I'm good to go... yeah it was driving me crazy I've had to look it up it's been going on for several weeks I'm glad I finally figured it out hope this helps you guys let me know @daby0729 

I'll try this. Problem is I've tried those for the 403 problem and they caused other issues eventually. But playback is so unreliable now that I might just do this for now.

**Comment by Heroshi1947** [comment]
hey everyone 
I tried this tweaking in developers options and now the app works perfectly 
go to settings > player > developer options > untick/uncheck all the fixes/patches just leave the 1st option which say something like "Premiuim user" checked on , restart app and this should work for you too.

**Comment by rpgdev** [comment]
I discovered that bypassing vpn for my ATV has gotten rid of the issue so far, did not get to apply any of the developer options suggested. Will check later if I had any left over from previous errors. It seems vpn breaks the new stable version when I used to use this app through vpn just fine.

**Comment by daniushko** [comment]
After the upgrade, the app on my TV works fine. I haven't done anything else to fix it.

**Comment by daby0729** [comment]
The tweaking that I did worked for me but I didn't notice some videos the sound was synced off so I just said it back to the way that it was and just left the fire stick frame drop too checked frame drop one I unchecked

**Comment by Nabukodonosor** [comment]
@yuliskov Can you fix that notifications screen issue?

**Comment by MYusufY** [comment]
> Published the beta with the latest fixes.
> https://github.com/yuliskov/SmartTube/releases/tag/26.07

Beta version actually worked fine for me!

**Comment by AlanSA-spec** [comment]
> Published the beta with the latest fixes. https://github.com/yuliskov/SmartTube/releases/tag/26.07

Excellent. Fixed the issue for me.

**Comment by yuliskov** [comment]
Prepared another build addressed 403 error but it's VERY UNSTABLE. Try it only if videos are fully unplayable.
https://github.com/yuliskov/SmartTube/releases/download/latest/SmartTube_beta_testing.apk

**Comment by CommanderRedYT** [comment]
I think this is quite weird/interesting when I can also use for example revanced without having this problem on my phone. Same network, same account, same IP.

**Comment by rpgdev** [comment]
> I think this is quite weird/interesting when I can also use for example revanced without having this problem on my phone. Same network, same account, same IP.

That's not weird, revanced is the official yt client with tweaks. smarttube it's a 3rd party client.

**Comment by CommanderRedYT** [comment]
> That's not weird, revanced is the official yt client with tweaks. smarttube it's a 3rd party client. 

Yes, but they are achieving the same thing: ad-free YouTube. So what I find interesting is which one gets blocked. Like, wouldn't the same steps that revanced uses to block them also be required for smarttube?

**Comment by rpgdev** [comment]
> Yes, but they are achieving the same thing: ad-free YouTube. So what I find interesting is which one gets blocked. Like, wouldn't the same steps that revanced uses to block them also be required for smarttube?

No, Revanced is the official client with tweaks. You literally have to download the official app and apply the tweaks afterwards. It's not even close to the same situation. Anyways, I'll ignore you now since this is off topic and spamming everyone's notifications.

**Comment by DadManBlues** [comment]
I just got the same issue a few minutes ago on version 27.37, Philips Android 11 tv.

I watched 3 hand-picked videos from the same channel. Since then it opens all the forthcoming videos one by one but doesn't play them. Is it became mandatory to sign in to SmartTube?

The message said 'Prove you're not a robot'. Why SmartTube is bothered if I'm a robot watching Yt videos? SmartTube is just an interface to Yt, isn't it? My watching habits don't generate any load on SmartTube servers, if they have any. Anyway, if I select videos manually on my remote, it should be enough to prove I'm not a robot.

**Comment by ixicalibur** [comment]
> I just got the same issue a few minutes ago on version 27.37, Philips Android 11 tv.
> 
> I watched 3 hand-picked videos from the same channel. Since then it opens all the forthcoming videos one by one but doesn't play them. Is it became mandatory to sign in to SmartTube?
> 
> The message said 'Prove you're not a robot'. Why SmartTube is bothered if I'm a robot watching Yt videos? SmartTube is just an interface to Yt, isn't it? My watching habits don't generate any load on SmartTube servers, if they have any. Anyway, if I select videos manually on my remote, it should be enough to prove I'm not a robot.

YOUTUBE is asking you to verify you are not a bot, not Smart tube.
